jni/rubicon.c

```diff
@@ -1037,7 +1041,13 @@ JNIEXPORT jobject JNICALL Java_org_beeware_rubicon_PythonInstance_invoke(JNIEnv
 
     PyObject *result;
     PyObject *pargs = PyTuple_New(3);
+    #if __SIZEOF_LONG_LONG__ == 8
+    PyObject *pinstance = PyLong_FromLongLong(instance);
+    #elif __SIZEOF_LONG__ == 8
     PyObject *pinstance = PyLong_FromLong(instance);
```

Comment:
I'm confused as to why instance (which is a jlong) can be of variable type - isn't jlong *always* a long long?

Reply:
A `jlong` is a signed 64-bit integer. See: https://docs.oracle.com/javase/7/docs/technotes/guides/jni/spec/types.html

Given that, it may be a `long long` on some systems, or a `long` on some systems. This ensures that we're not screwing it up. Happy to consider other approaches.